hbase-client/src/main/java/org/apache/hadoop/hbase/shaded/protobuf/ProtobufUtil.java

```diff
@@ -2192,6 +2194,9 @@ public static String getShortTextFormat(Message m) {
       ClientProtos.CoprocessorServiceRequest r = (ClientProtos.CoprocessorServiceRequest) m;
       return "coprocessorService= " + r.getCall().getServiceName() + ":"
         + r.getCall().getMethodName();
+    } else if (m instanceof MasterProtos.MoveRegionRequest) {
+      // This should be small and safe to output. No data.
```

Comment:
The "no data" clause doesn't really make sense here.

Reply:
Thanks. Yeah, this is a copy-paste error. I'll remove it.